anthias_app/views.py

```diff
@@ -0,0 +1,35 @@
+from django.shortcuts import render, HttpResponse
+from os import getenv
+from settings import settings
+from urllib.parse import urlparse
+
+import logging
+
+from lib.utils import (
+    is_balena_app,
+    is_demo_node,
+)
+
+
+def index(request):
+    player_name = settings['player_name']
+    my_ip = urlparse(request.build_absolute_uri()).hostname
+    is_demo = is_demo_node()
+    resin_uuid = getenv("RESIN_UUID", None)
```

Comment:
Let's rename all resin references to balena.

Reply:
As this PR is significantly bulky at the moment, this can be resolved in Screenly/Anthias#1776.